test/paper_tooltip_test.dart

```diff
@@ -0,0 +1,206 @@
+@TestOn('browser')
+library polymer_elements.test.paper_tooltip_test;
+
+import 'dart:async';
+import 'dart:html';
+import 'package:polymer_elements/paper_tooltip.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+import 'sinon/sinon.dart' as sinon;
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-tooltip/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  bool isHidden(element) {
```

Comment:
I thought you suggested creating global matcher in common.dart.
For local purpose method looks fine and as you said, I'd prefer to keep it simple as it is.